src/transaction.rs

```diff
@@ -8,12 +8,17 @@ use rayon::prelude::*;
 use signature::{KeyPair, KeyPairUtil, PublicKey, Signature, SignatureUtil};
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
-pub struct Transaction {
+pub struct Signed {
```

Comment:
Last review asked that you change this to TransactionData and that you remove 'from' from it.